s3/pkg/datastore/azure/azure.go

```diff
@@ -146,34 +146,17 @@ func (ad *AzureAdapter) Get(ctx context.Context, object *pb.Object, start int64,
 	log.Infof("get object[Azure Blob], bucket:%s, objectId:%s\n", bucket, object.ObjectId)
 
 	blobURL := ad.containerURL.NewBlobURL(object.ObjectId)
-	log.Infof("blobURL:%v, size:%d\n", blobURL, object.Size)
-
-	var buf []byte
-	if end < object.Size - 1 {
```

Comment:
Didn't get this part of the code, why are we removing this codition

Reply:
As blobURL.Download can take care of all situations, small size and big size, no need to download to buffer first, and then return.

Reply:
Ok